client/components/filters/advanced/index.js

```diff
@@ -131,46 +133,47 @@ class AdvancedFilters extends Component {
 
 	getUpdateHref( activeFilters, matchValue ) {
 		const { path, query, config } = this.props;
-		const updatedQuery = getQueryFromActiveFilters( activeFilters, query, config );
+		const updatedQuery = getQueryFromActiveFilters( activeFilters, query, config.filters );
 		const match = matchValue === 'all' ? undefined : matchValue;
 		return getNewPath( { ...updatedQuery, match }, path, query );
 	}
 
+	isEnglish() {
+		const { siteLocale } = wcSettings;
+		return /en-/.test( siteLocale );
+	}
+
```

Comment:
Determining if the language is English here so that fields can be lined up vertically to match the designs:

![screen shot 2018-08-01 at 12 12 59 pm](https://user-images.githubusercontent.com/1922453/43493899-5253bb72-9584-11e8-9e6e-62e6d7756000.png)

This can only happen reliably when the language is known to be English.

Reply:
Maybe we can do the CSS a little differently (in another PR) to remove this restriction (I'm thinking maybe a combo of min-width and flex).

Reply:
I can open a new issue to look into this further. What I have, flex + width: 100%, on the input field ensures that the input takes up the maximum space, and that is working well.

The `isEnglish` check is helpful for the defined width of the first column on the left, eg "Order Status", "Product", "Coupon Code", etc. Without knowing the sentence structure, it would look odd to define that width explicitly.